SConstruct

```diff
@@ -85,7 +85,7 @@ options.Add(
 options.Add(
 	"CXXFLAGS",
 	"The extra flags to pass to the C++ compiler during compilation.",
-	[ "-pipe", "-Wall", "-Werror", "-O3" ]
+	[ "-pipe", "-Wall", "-Werror", "-O3", "-ffast-math" ]
```

Comment:
Do you have any numbers for the improved performance? And since the filters are compiled into OIIO, do you know if we would benefit much from enabling it there?